runtime/src/weights/pallet_sidechain.rs

```diff
@@ -0,0 +1,39 @@
+//! Autogenerated weights for `pallet_teerex`
```

Comment:
@clangenb Should we keep the weights of a test-node up to date, or can be we leave it like this?

Reply:
We can leave it like this, as long as we did not change the logic of the functions drastically, the weights will stay similar anyhow.